addons/stock/wizard/make_procurement_product.py

```diff
@@ -36,6 +38,30 @@ def onchange_product_id(self, cr, uid, ids, prod_id):
         'qty': lambda *args: 1.0,
     }
 
+    def fields_view_get(self, cr, uid, view_id=None, view_type='form', context=None, toolbar=False, submenu=False):
```

Comment:
@tde-banana-odoo, you are right,
It is possible to store product_tmpl_id and variant_count directly on the wizard instead of hacking the fields view get.